flask_seasurf.py

```diff
@@ -104,6 +104,7 @@ class SeaSurf(object):
     def __init__(self, app=None):
         self._exempt_views = set()
         self._include_views = set()
+        self._exempt_urls = tuple()
```

Comment:
I see. It's probably fine to use a tuple, I only mentioned it since we use sets for the views.